### DISCUSSION TITLE
Community Policy Library

### DISCUSSION THREAD

**Discussion by mkamadeus** [open-policy-agent/community#237] [Policy Exchange]
Hello, I was wondering whether something like a community policy library/gallery/repository exists? Would be really helpful in common checks. E.g:

- A policy that ensures S3 buckets are private in a Terraform plan
- A policy that ensures S3 buckets are encrypted
- A policy that ensures limits for pods are set below a certain threshold
- etc.

Policy bundling will be up to the user. The repository/gallery will serve only as an example an for use.

Excellent project by the way 😄

**Comment by anderseknert** [comment]
Hey there! A few policy packages linked in the [Awesome OPA](https://github.com/anderseknert/awesome-opa#policy-packages) list. I'd be happy to add more :)

**Reply by mkamadeus** [reply]
That's cool! I expected something more of like this https://github.com/Scalr/sample-tf-opa-policies that is listed there which have fine-grained policies, would be cool if there exists a repository dedicated for fine-grained policies that we can (maybe) bundle ourselves

**Reply by anderseknert** [reply]
There are a few of those listed there as well. Did you check the [Infrastructure as Code](https://github.com/anderseknert/awesome-opa#infrastructure-as-code) section? Scalr is listed as well, but they did not submit their sample repo. I'll make a note to have it added. Thanks!
